test/unit/edition/topic_test.rb

```diff
@@ -9,6 +9,10 @@ class EditionWithTopics < Edition
     @topic = create(:topic)
   end
 
+  test "includes PublishesToPublishingApi" do
+    assert Topic.new.is_a?(PublishesToPublishingApi)
+  end
```

Comment:
Is it worth testing the include? Is there some effect we could test that is only provided by including this module? (e.g. that it publishes when the record is saved)